src/components/workspace-utils.js

```diff
@@ -161,8 +165,7 @@ export const SnapshotInfo = ({
   })
 
   const tdrErrorDisplay = () => div({ style: { paddingLeft: '1rem' } }, [
-    div({ style: Style.dashboard.header }, ['Error']),
-    `Details of this snapshot could not be loaded due to a problem calling Terra Data Repo: ${JSON.stringify(snapshotLoadError)}`
+    div({ style: Style.dashboard.header }, ['Error']), snapshotLoadError
```

Comment:
[Nit] When `children` are given on a newline, we tend to delineate each child by new lines:

```suggestion
    div({ style: Style.dashboard.header }, ['Error']),
    snapshotLoadError
```